app/sender/messageSender.go

```diff
@@ -72,6 +81,23 @@ func SendMessage(s *store.Session, m *store.Message) (*store.Message, error) {
 		}
 	}
 	var recipient string
+	// check if user uuid exists and if yes send it to the uuid instead of the phone number
+	if s.UUID != emptyUUID && s.UUID != "" && !s.IsGroup {
+		recipient = s.UUID
+		index := strings.Index(recipient, "-")
+		if index == -1 {
+			recipient = HexToUUID(recipient)
+		}
+	} else {
+		recipient = s.Tel
```

Comment:
Just a question, why is there new code for sending using the tel number? As Axolotl currently is capable of sending message, I guess that a similar code exists somewhere else. I should probably be removed then?